fiftyone/zoo/models/__init__.py

```
@@ -620,7 +623,18 @@ class RemoteZooModelsManifest(ZooModelsManifest):
     _MODEL_CLS = RemoteZooModel
 
 
+@functools.lru_cache(maxsize=1)
 def _load_zoo_models_manifest():
```

Ruff (0.16.3)

[warning] Missing return type annotation for private function `_load_zoo_models_manifest`

(ANN202)